### DISCUSSION TITLE
mark files & see calcsize of selected files

### DISCUSSION THREAD

**Discussion by kristoferus75** [gokcehan/lf#1214] [General]
Hi !

Is it possible that you can mark files and see a sum (GB) of the selected files ?

Thanks !

**Comment by ilyagr** [comment]
I don't think so. I highly recommend https://dev.yorhel.nl/ncdu.

**Update:** I didn't read your question carefully enough, sorry. This is possible as @joelim-work described. `ncdu` is great if you want to get an interactive browser for directory sizes (but it's a separate tool).

**Comment by joelim-work** [comment]
If you want to just simply select some files/directories and display the total size, you can run a shell command inside `lf`:

```
!du -ch $fx
```

If you only want the total size, then the output is exactly one line and can be displayed in the status bar:

```
%du -ch $fx | tail -n 1 | cut -f 1
```

- `!` runs a shell command and waits for a keypress
- `%` runs a shell command and pipes the output to the status bar
- `fx` is a variable containing the selected file(s)

These shell commands can also be assigned a keybinding using the `map` command for convenience.

**Reply by kristoferus75** [reply]
Thanks a lot :-)
